server/java/README.md

```diff
@@ -81,31 +72,55 @@ Successfully built e9cb70abb481
 [main] INFO com.godaddy.asherah.grpc.AppEncryptionConfig - revoke check minutes set to = 60 minutes
 [main] INFO com.godaddy.asherah.grpc.AppEncryptionServer - server has started listening on /tmp/appencryption.sock
 ```
+## Development notes: 
+* Make sure you have JAVA 1.8
```

Comment:
These should not be moved down. These conditions need to be fulfilled before one can run `mvn clean install` on the java-server

Reply:
This was done to make the documentation consistent with the other READMEs. I've added a note about JAVA 1.8 in the `Running the server` section now. The AWS files are really needed only for the unit tests and so should be in a separate section, similar to other READMEs.